motion-led-controller/README.md

```diff
@@ -0,0 +1,18 @@
+# openwrt-ckt-feeds
+
+Feeds for CreatorKit packages.
+
+Package           | Description
+:---------------- | -----------------------------
+button-gateway    | Button gateway is an application running on Ci40 which acts as a gateway for MikroE boards.
+button-led-controller    | Button-Led controller is an application for controlling LED on Ci40 as per button-press on Clicker boards.
+device-manager    | Device manager is an application for provisioning devices with access details for FlowCloud.
+libflow           | Binary package for FlowLibraries.
+motion-led-controller  | Motion Led controller is an application running on Ci40 which controls the LED based on the motion sensor events from MikroE board.
+webscripts        | Webscripts to provision Gateway and Constrained devices.
+
+Command                                         | Description
+:---------------------------------------------- | :---------------------------------------
+```$ ./scripts/feeds/install device-manager```  | Install only the package device-manager
+
+
```

Comment:
Why are there 2 different README.md files? there should not be a motion-led-controller/README.md in openwrt-ckt-feeds